src/main/java/dk/kb/present/View.java

```diff
@@ -85,30 +88,55 @@ public MediaType getMime() {
     }
 
     @Override
-    public String apply(String recordID, String content, String child) {
-        final Map<String, String> metadata = new HashMap<>();
-        metadata.put("recordID", recordID);
-        metadata.put("origin", origin);
+    public String apply(DsRecordDto record) {
+        final Map<String, String> metadata = createBasicMetadataMap(record);
 
-        if (child != null ) {
-            metadata.put("childRecord", child);
-        }
+        String content = record.getData();
+
+        switch (strategy) {
+            case "manifestation":
+                updateMetadataMapWithPreservicaManifestation(record, metadata);
+            case "none":
 
+        }
 
         for (DSTransformer transformer: this) {
             try {
                 content = transformer.apply(content, metadata);
             } catch (Exception e) {
                 String message = String.format(
                         Locale.ROOT, "Exception in View '%s' while calling %s with recordID '%s' and metadata %s",
-                        getId(), transformer, recordID, metadata);
+                        getId(), transformer, record.getId(), metadata);
                 log.warn(message, e);
                 throw new InternalServiceException(message);
             }
         }
         return content;
     }
 
+    /**
+     * Extract metadata which should always be present to a metadata map.
+     * @param record to extract basic metadata from.
+     */
+    private Map<String,String> createBasicMetadataMap(DsRecordDto record) {
+        final Map<String, String> metadata = new HashMap<>();
+        metadata.put("recordID", record.getId());
+        metadata.put("origin", origin);
+        return metadata;
+    }
+
+    /**
+     * Update the map of metadata with child record from the deliverable unit contained in the input record.
+     * @param record with an expanded tree containing manifestation childs.
+     * @param metadata map that values from the record is extracted to.
+     */
+    private void updateMetadataMapWithPreservicaManifestation(DsRecordDto record, Map<String, String> metadata) {
+        String child = getFirstChild(record);
+        if (child != null ) {
+            metadata.put("childRecord", child);
```

Comment:
As we are very specific here, maybe call it `manifestation` instead of `childRecord`?